implementation/test.py

```diff
@@ -0,0 +1,41 @@
+# KOD: Marta Rybarczyk
+
+from datetime import datetime
+from implementation.Implementation import Implementation
+
+class Test:
+    def test_LD(self, i):
+        for lamb in range(1, 10):
+            for d in range(1, 6):
+                l = lamb / 10

```

Comment:
lamb / 10? nie może być lambda powyżej 1?

Reply:
O to w sumie trzeba się dopytać kogoś, bo nie wiem sam

Reply:
nie pamiętam teraz jak to ujął, ale jak rozmawiałam z ŁK to mówił, że to nie ma być duże, czy coś takiego

Reply:
dlatego robiłam tak